source/docs/05-mensural.xml

```diff
@@ -397,6 +397,22 @@
                         <specDesc key="att.plica.vis" atts="dir len"/>
                      </specList>
                   </p>
+                  <figure>
+                     <head>Encoding and rendition of plicas</head>
+                     <graphic url="../images/modules/mensural/ex_plicae.png"/>
+                  </figure>
+               </div>
+               <div xml:id="dots" type="div3">
+                  <head>Dots</head>
+                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element. This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
```

Comment:
```suggestion
                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element. This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
```

My understanding is that dots of division and augmentation alter the duration of notes and rests before, but please correct me if I'm wrong. But I think it's better to not refer to "the original source" and instead focus on describing the effect of the dot on the notes or rest. (Otherwise it just sounds like you're saying "place the dot where the dot should be placed", which isn't very helpful for describing what it does or why it's there.) 

You don't actually explain what the difference is between the different types of dots, or how they function, anywhere in this section.

Reply:
Well, the augmentation dot definitely changes the duration of the note preceding it (because this dot adds half of the note's value). However, that is not the case for the dot of division. A dot of division changes the way you would interpret the notes close to it, but not necessarily the note preceding it. For example, if you have `breve, semibreve, semibreve, semibreve, breve` you would keep the first breve perfect and everything else will have a regular value. If you add a dot between the first and second semibreve (i.e., `breve, semibreve, dot, semibreve, semibreve, breve`), then the first breve will be imperfected by the following semibreve, and the last (the third) semibreve will be altered. The dot of division changes the perfect groupings of notes and, therefore, the interpretation of one or more notes in the sequence.

Reply:
> But I think it's better to not refer to "the original source" and instead focus on describing the effect of the dot on the notes or rest. (Otherwise it just sounds like you're saying "place the dot where the dot should be placed", which isn't very helpful for describing what it does or why it's there.)

Yes, that is what I am saying, because we usually place the dots of a dotted note (which is basically also an augmentation dot) as a child or attribute of that note element. So I want to make it clear where to place it, just as the sibling following that note. Is there a better way to make this clear?

Reply:
> You don't actually explain what the difference is between the different types of dots, or how they function, anywhere in this section.

I am trying to clarify this and point to the examples in the commit [a4d1604](https://github.com/music-encoding/music-encoding/pull/1191/commits/a4d1604f4dbbac17f5649e8037fe2bd4d02e2a51)